tests/e2e/travis-e2e.sh

```diff
@@ -8,12 +8,14 @@ set -u
 # print each command before executing it
 set -x
 
+K8S_VERSION="${K8S_VERSION:-"1.19.0"}"
```

Comment:
The kind recommendation regarding image versions is to include both the version and the digest ([ref](https://github.com/kubernetes-sigs/kind/releases/tag/v0.9.0)). I think we should start following this and thus change the variable name to something like `KIND_IMAGE` to be a bit more precise.

Reply:
Right now this doesn't change anything as `kind` does not produce more than one image/manifest with a tag. It would be important to pin to SHA if they would have more than one manifest associated with a tag.

Source: https://hub.docker.com/r/kindest/node/tags